Source/ACE.Server/Factories/LootGenerationFactory.cs

```diff
@@ -88,12 +86,34 @@ public static List<WorldObject> CreateRandomLootObjects(TreasureDeath profile)
 
             if (itemChance <= profile.MagicItemChance)
             {
+                int aetheriaDropChance;
+                bool aetheriaGenerated = false;
                 numItems = ThreadSafeRandom.Next(profile.MagicItemMinAmount, profile.MagicItemMaxAmount);
                 for (var i = 0; i < numItems; i++)
                 {
-                    lootWorldObject = CreateRandomLootObjects(profile.Tier, true, lootBias);
-                    if (lootWorldObject != null)
-                        loot.Add(lootWorldObject);
+                    // Coalesced Aetheria don't drop for tiers less than 5
+                    // According to wiki, Weapon Mana Forge chests don't drop Aetheria
+                    // Otherwise, 50% chance to drop a Coalesced Aetheria 
+                    if (aetheriaGenerated == false && profile.Tier > 4 && lootBias != LootBias.Weapons)
+                        aetheriaDropChance = ThreadSafeRandom.Next(1, 2);
```

Comment:
Is there a particular reference for the 50% drop rate for Aetheria?

This seems like it would be an extremely high drop rate...

Reply:
I haven't found any reference to the amount that it drops.  There are statements to where it won't drop.

The 50% chance is for each roll in the magic loot category to get either an Aetheria or other magic containing item.   Only one is allowed to generate per call into the loot generation, so even on multiple rolls, only a max of one Aetheria will be generated.  While play testing this patch in multiple loot tiers, the Aetheria spawned less than the Summon Essense do.  With any less chance to generate an Aetheria, it felt too sparse.  At any rate, it is just the first stab at getting them into the mix.  If others that may have an even better sense of it say its still too high, it can be decreased.

Reply:
I did make a generalization and inference from a note in the wiki for Steel Chests, in that either Coalesced Aetheria or a weapon drops for its two non-scroll or mana charge slots.

Reply:
I am certainly open to changing it, if the drop rate still feels to high.

Reply:
i can try playtesting when i get a chance

knee-deep in bug fixes for the live server atm...